### PR TITLE
[Planning]: [Autotuning], add autotuning raw feature generator interf…

### PR DESCRIPTION
**Pull request by fanhaoyang002**
…ace and build files

### CODE REVIEW

File: modules/planning/tuning/autotuning_raw_feature_generator.h [hunk 1]
```diff
@@ -0,0 +1,70 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
```
Comment (startcode) [inline modules/planning/tuning/autotuning_raw_feature_generator.h]: 2018

File: modules/planning/tuning/autotuning_raw_feature_generator.h [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+#include "modules/planning/common/reference_line_info.h"
+#include "modules/planning/proto/auto_tuning_raw_feature.pb.h"
+namespace apollo {
+namespace planning {
+
+class AutotuningRawFeatureGenerator {
+public:
+
+	// @brief class constructor
+	AutotuningRawFeatureGenerator() = default;
```
Comment (startcode) [inline modules/planning/tuning/autotuning_raw_feature_generator.h]: recommend to reformat your code with clang-format

File: modules/planning/tuning/autotuning_raw_feature_generator.cc [hunk 3]
```diff
@@ -0,0 +1,43 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
+ *
```
Comment (jinghaomiao) [inline modules/planning/tuning/autotuning_raw_feature_generator.cc]: 2018

File: modules/planning/tuning/autotuning_raw_feature_generator.cc [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+common::Status AutotuningRawFeatureGenerator::evaluate_trajectory(
+    const common::TrajectoryPoint& trajectory_point,
+    const ReferenceLineInfo& reference_line_info, const Frame& frame,
+    autotuning::TrajectoryPointRawFeature* const trajectory_point_feature)
+    const {
+  return common::Status::OK();
+}
+
+}  // namespace planning
+}  // namespace apollo
```
Comment (jinghaomiao) [inline modules/planning/tuning/autotuning_raw_feature_generator.cc]: Can you please provide the unit test as well? We'll be enforcing unit test coverage.

**Pull request merged**